common/components/common/volunteers/VolunteerSection.jsx

```diff
@@ -295,6 +296,7 @@ class VolunteerSection extends React.PureComponent<Props, State> {
                     key={i}
                     volunteer={volunteer}
                     isProjectAdmin={this.props.isProjectAdmin}
+                    isProjectCoOwner={this.props.isProjectCoOwner && !(header === "CO-OWNERS")}
```

Comment:
"CO-OWNERS" should be a constant if we're using it in multiple places

Reply:
We're using it 2 times within a few lines of each other. Does that rate adding it to ProjectConstants?

Reply:
Got it.  Done.